src/current/releases/v19.2.md

```diff
@@ -0,0 +1,13 @@
+---
+title: What&#39;s New in v19.2
+toc: true
+toc_not_nested: true
+summary: Additions and changes in CockroachDB version v19.2 since version v19.1
+major_version: v19.2
+docs_area: releases
+keywords: gin, gin index, gin indexes, inverted index, inverted indexes, accelerated index, accelerated indexes
+---
+
+<a name="v19-2-0"></a>
+
+This release is no longer supported. For more information, see our [Release support policy]({% link releases/release-support-policy.md %}).
```

Comment:
TODO: @kenliu-crl suggests linking to the archived versions page so users can download the archived docs